package/com.unity.formats.usd/Tests/Editor/ExportTests.cs

```diff
@@ -177,6 +179,13 @@ public void ExportSkeletalAnimation_ExportedPrimsHaveSkeletalTypes()
             var skeletonInUsdType = skeletonInUsd.GetTypeName();
             var expectedSkeletonInUsdType = new pxr.TfToken("Skeleton");
             Assert.AreEqual(expectedSkeletonInUsdType, skeletonInUsdType);
+
+            // check for SkelBindingAPI/MaterialBindingAPI
+            var primInUsd = m_USDScene.GetPrimAtPath(new pxr.SdfPath(UnityTypeConverter.GetPath(mesh.transform)));
+            var skelBindingAPIType = pxr.TfType.FindByName("SkelBindingAPI");
+            Assert.IsTrue(primInUsd.HasAPI(skelBindingAPIType));
```

Comment:
Could you add a failure message to the assert to make it easier to debug please :)